### ISSUE TITLE
Question on boost.py: Why runtime instructions only copies boost-binary?

### ISSUE THREAD

**Issue by qlibp** [NVIDIA/hpc-container-maker#460]
As boost relies on zlib etc, which is by default a dynamic-build, why we don't need to copy the ospackages stuff?

**Comment by samcmill** [comment]
Yes, there probably should be a set of runtime ospackages that are installed.  Besides zlib, is there anything else you've found that should be installed?

**Comment by qlibp** [comment]
I do not run hpccm actually. I just got this question by inspecting the code. Seems that most of the building blocks do not copy the ospackages im the runtime call. Am I missing something？
